Ark.Tools.FtpClient.ArxOne/FtpClientPoolArxOne.cs

```diff
@@ -13,30 +13,35 @@
 
 namespace Ark.Tools.FtpClient
 {
-
     public class FtpClientPoolArxOne : FtpClientBase, IFtpClientPool
     {
         private readonly ArxOne.Ftp.FtpClient _client;
         private readonly SemaphoreSlim _semaphore;
+        private readonly Action<FtpClientParameters>? _configurer;
 
         private bool _isDisposed  =  false;
 
-        public FtpClientPoolArxOne(int maxPoolSize, FtpConfig ftpConfig)
+        public FtpClientPoolArxOne(int maxPoolSize, FtpConfig ftpConfig, Action<FtpClientParameters>? configurer = null)
             : base(ftpConfig, maxPoolSize)
         {
             _client = _getClient();
+            _configurer = configurer;
```

Comment:
doesn't the _getClient() use rhe _configurer?

anw in general, assignments before initialization function